ci/provisioning/post_provision_config_nodes.sh

```diff
@@ -81,12 +81,17 @@ mkdir -p /localhome
 if ! grep ":$MY_UID:$MY_UID:" /etc/passwd; then
   useradd -b /localhome -g "$MY_UID" -u "$MY_UID" -s /bin/bash jenkins
 fi
-mkdir -p /localhome/jenkins/.ssh
-cat /tmp/ci_key.pub >> /localhome/jenkins/.ssh/authorized_keys
-cat /tmp/ci_key.pub >> /root/.ssh/authorized_keys
-mv /tmp/ci_key.pub /localhome/jenkins/.ssh/id_rsa.pub
-mv /tmp/ci_key /localhome/jenkins/.ssh/id_rsa
-mv /tmp/ci_key_ssh_config /localhome/jenkins/.ssh/config
+jenkins_ssh=/localhome/jenkins/.ssh
+mkdir -p "${jenkns_ssh}"
```

Comment:
(lint) jenkns_ssh is referenced but not assigned (did you mean 'jenkins_ssh'?). [SC2154]